packages/nodes-base/nodes/HttpRequest/HttpRequest.node.ts

```diff
@@ -0,0 +1,27 @@
+import {
+	INodeTypeBaseDescription,
+	INodeVersions,
+} from 'n8n-workflow';
+
+import { HttpRequestV1 } from './HttpRequest_v1.node';
+import { HttpRequestV2 } from './HttpRequest_v2.node';
+import { NodeVersionedType } from '../../src/NodeVersionedType';
+
+export class HttpRequest extends NodeVersionedType {
```

Comment:
just realized this might end up being a breaking change to how we read data for the website. because we read properties from this node entry file. we should probably test this out